packages/@aws-cdk/aws-redshift/lib/database-props.ts

```diff
@@ -0,0 +1,26 @@
+import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
+import { ICluster } from './cluster';
+
+/**
+ * Properties for accessing a Redshift database
+ */
+export interface DatabaseProps {
```

Comment:
We usually reserve `Props` for construct initialization argument. For structs that are supertypes of a `Props` interface, we use `Options` instead.

```suggestion
export interface DatabaseOptions {
```